### PR TITLE
Updated all instances of "Sign in with Twitter" to "Sign in with X" in SignIn Page TestScripts

### PR DESCRIPTION
**Pull request by lavishaGit**
### What github.com/wevote/WebApp/issues does this fix?
Fix is in  SignIn page change in TwitterButton text from "Sign in with Twitter" to "Sign in with X"

### Changes included this pull request?
Updated few locaters on Signin Page for more stable and straightforward
Updated test script with require validation for "Sign in with X" instead of "SignIn Twitter"

### CODE REVIEW

File: tests/browserstack_automation/page_objects/signin.page.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ class SignIn extends Page{
     }
 
     get signInWithTwitterTextElement(){
-        return $('//span[text()="Sign in with Twitter"]');
+        return $('.csbvaL');
```
Comment (fdiazq) [inline tests/browserstack_automation/page_objects/signin.page.js]: Could you locate the element with an id instead of class name?

**Review by fdiazq — CHANGES_REQUESTED** [review]

File: tests/browserstack_automation/page_objects/signin.page.js (hunk 1 above)
Comment (lavishaGit) [reply tests/browserstack_automation/page_objects/signin.page.js]: Thanks for reviewing the changes, @fdiazq! I tried adding the ID for the "Sign in with Twitter" element, but  the WebApp running on localhost port 3000 is not displaying the Sign in with X/Twitter option on the SignIn page. As a result, I opted for the class name strategy, which is more stable .
Please have a look .

**Comment by lavishaGit** [comment]
@fdiazq @DaleMcGrew  I updated the changes that are requested for SignIn page .Please review it .

**Review by fdiazq — APPROVED** [review]
@lavishaGit  👍

**Pull request merged**